src/app/containers/Timestamp/__snapshots__/index.test.jsx.snap

```diff
@@ -0,0 +1,107 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`Timestamp should add prefix and suffix 1`] = `
+.c0 {
+  font-size: 0.875rem;
+  line-height: 1rem;
+  color: #757575;
+  display: block;
+  font-family: ReithSans,Helvetica,Arial,sans-serif;
+  padding-bottom: 0.25rem;
+}
+
+.c0:last-child {
+  padding-bottom: 1rem;
+}
+
+@media (min-width:20rem) and (max-width:37.4375rem) {
+  .c0 {
+    line-height: 1.125rem;
+  }
+}
+
+@media (min-width:37.5rem) {
+  .c0 {
+    font-size: 0.8125rem;
+  }
+}
+
+<time
+  className="c0"
+  dateTime="2018-10-19"
+>
+  It was 
+  192 days ago
```

Comment:
Are we certain this isn't going to change with the date? EG: tomorrow being "193 days ago"

Reply:
Changed the test so that the date used remains constant, and doesn't change with the date.